### ISSUE TITLE
Docker container exits after command

### ISSUE THREAD

**Issue by eikooc** [stripe/stripe-cli#497]
_The more information we have the easier it is for us to help. Feel free to remove any sections that might not apply_
Trying to use the CLI with Docker, but it can only execute one command and then exits.

### Issue
_Describe what happened and what you were trying to do_
Trying to use the CLI with Docker to test webhooks.

### Expected Behavior
_Tell us what you expected to happen_
The Docker container running the CLI should not exit, it should stay running.

### Steps to reproduce
_What are the steps we can take to reproduce this and verify it's fixed?_
Try running `docker run --rm -it stripe/stripe-cli latest`.

See that there is an error:
```
Unknown command "latest" for "stripe".
See "stripe --help" for a list of available commands.
```

Run any other command for example `login` instead of `latest` and see the container exit after running.

### Traceback
_Share any debug output that was given by the CLI_

### Environment
_Select one of: macOS, Linux, Windows, Other_
Docker on Linux

**Comment by richardm-stripe** [comment]
Thanks for pointing this out @eikooc! This is a bug in our docs -- the command should be
```shell
docker run --rm -it stripe/stripe-cli:latest
```
with a colon before 'latest' instead of a space... `latest` is a tag on docker that corresponds to the latest version of the stripe-cli and is not meant to be a stripe-cli command.

We will look into getting the docs fixed.

**Comment by eikooc** [comment]
Hi @richardm-stripe. That sounds good, but it only solves half the problem.

You still need to make the docker container stay alive so I can attach to it :+1:

**Comment by richardm-stripe** [comment]
Yes, I see how this is confusing -- after the first step the docs sort of assume that you are inside a shell with access to `stripe` in the path.

You've got a couple options:
1. You can override the docker container's "entrypoint" to be a shell rather than the stripe-cli itself and that will plop you into a shell and allow you to run `stripe login` and then e.g. `stripe listen` or another command in a single shell session, without the session terminating after each command, i.e.
```shell
docker run --rm --entrypoint /bin/sh -it stripe/stripe-cli:latest
$ stripe login
...
$ stripe <some-other-command>
```
but with this solution you will need to run `stripe login` at the beginning of each session.

2. You can mount a docker "volume" to store the results of `stripe login` on the host -- in your home directory or something -- and use them from subsequent docker invocations, i.e.

```shell
$ docker run --rm -v ~/.config/stripe:/root/.config/stripe -it stripe/stripe-cli:latest login
...
$ docker run --rm -v ~/.config/stripe:/root/.config/stripe -it stripe/stripe-cli:latest <some-other-stripe-cli-command>
```

**Comment by joncar** [comment]
Came here to report this issue with persisting the login information. I also suggest adding `--network host` to make the listen command easier to use from inside the Docker container.

**Comment by kerbe** [comment]
Came here while I was trying to figure out how to persist login information. 
There is mention about using volume in this page in docs: https://stripe.com/docs/cli/docker
But it doesn't tell what kind of volume/where it should be mounted.

**Comment by hirokihokari** [comment]
Could use the [`STRIPE_API_KEY`](https://stripe.com/docs/cli/docker) env var though. That seems to be the simplest solution to this?

**Comment by nfacciolo** [comment]
I

> Could use the [`STRIPE_API_KEY`](https://stripe.com/docs/cli/docker) env var though. That seems to be the simplest solution to this.

According to [stripe cli doc](https://stripe.com/docs/cli/api_keys), it's possible:

![image](https://user-images.githubusercontent.com/10351721/227153662-dadfd919-5d08-4018-8cea-fac2e7994e19.png)
